docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -74,12 +74,12 @@ If you log in with GitHub, Codacy requires the following [app permissions](https
     <tr>
       <td>Webhooks</td>
       <td>Read & Write</td>
-      <td>Codacy creates webhooks for organization and repository events (creation, deletion, member added, etc.) to enable <a href="../../organizations/what-are-synced-organizations/">synced organizations</a>.</td>
+      <td>Codacy creates webhooks for organization and repository events (creation, deletion, member added, etc.) to enable <a href="../../organizations/what-are-organizations/">organizations</a>.</td>
     </tr>
     <tr>
       <td>Members</td>
       <td>Read Only</td>
-      <td>Codacy retrieves information about organization members and teams to enforce permissions, enable synced organizations and user management.</td>
+      <td>Codacy retrieves information about organization members and teams to enforce permissions, enable organizations and user management.</td>
```

Comment:
Are we missing the link here?

```suggestion
      <td>Codacy retrieves information about organization members and teams to enforce permissions, enable Codacy <a href="../../organizations/what-are-organizations/">organizations</a> and user management.</td>
```

Reply:
Same as above, I think we should actually drop all these "... to enable organizations" bits. :thinking:

Reply:
Right. Like this?

```suggestion
      <td>Codacy retrieves information about organization members and teams to enforce permissions and user management.</td>
```